amplify-tools/amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/AmplifyTools.groovy

```diff
@@ -5,228 +5,230 @@ import groovy.json.JsonSlurper
 import groovy.json.JsonOutput
 
 class AmplifyTools implements Plugin<Project> {
-	void apply(Project project) {
-		def doesNodeExist = true
-		def doesGradleConfigExist
-
-		// profile name can be changed in amplify-gradle-config
-		def profile = 'default'
-		def accessKeyId = null
-		def secretAccessKey = null
-		def region = null
-		def envName = 'amplify'
-		def syncEnabled = 'true'
-
-		project.task('verifyNode') {
-			try {
-				project.exec {
-					commandLine 'npx', '-v'
-					standardOutput = new ByteArrayOutputStream()
-				}
-			} catch (commandLineFailure) {
-				doesNodeExist = false
-				println("Node is not installed. Visit https://nodejs.org/en/download/ to install it")
```

Comment:
Maybe we should say the npx was not found instead since as node could be installed without npx